ROLES.md

```diff
@@ -232,31 +232,44 @@ approver in an OWNERS file:
 
 ## Lead
 
-Working group leads, or just ‘leads’, are approvers of an entire area that have
-demonstrated good judgement and responsibility. Leads accept design proposals
-and approve design decisions for their area of ownership.
+Working group leads, or just ‘leads’, coordinate working group processes and
+decision-making. Leads are responsible for selecting appropriate processes and
+tracking and reporting on the working group's results to the TOC. Leads
+collaboratively take responsibilty for accepting design proposals and approving
+design decisions in their area of ownership.
+
+Working group leads may or may not overlap with code approvers -- the
+expectation is that working group leads are responsible for the overall quality
+of the working group's product (including documentation, testing, and
+usability), rather than just the service's code.
 
 ### Requirements
 
 Getting to be a lead of an existing working group:
 
 - Recognized as having expertise in the group’s subject matter.
 
-- Approver for some part of the codebase for at least 3 months.
+- Substantial contributions to the working group. Note that these contributions
+  may include design documents, documentation, improvements to the working group
+  process, or other non-code work which has improved the working group's
+  deliverables.
 
-- Member for at least 1 year or 50% of project lifetime, whichever is shorter.
+- Member for at least 4 months.
 
-- Primary reviewer for 20 substantial PRs.
+- Sponsored by an existing Working Group lead or a majority of active working
+  group contributors.
 
-- Reviewed or merged at least 50 PRs.
+- Approved by the technical oversight committee.
```

Comment:
We need to have a process by which a non-googler can be on the TOC, otherwise this can be read as "Approved by Google".

Reply:
I 100% agree that we need to describe the process for selecting TOC members.

I expect that this will be a longer discussion; the current `TECH-OVERSIGHT-COMMITTEE.md` file says:

> The members of the TOC are shown below. Membership in the TOC is determined by
> the [Steering committee](./STEERING-COMMITTEE.md).

But does not provide specific details about how this occurs.

Reply:
The actual change here is:

    - - Sponsored by the technical oversight committee.
    + - Approved by the technical oversight committee.

(i.e. a change from "sponsored" to "approved" because that seemed to make the process more explicit)